gen3-client/jwt/functions.go

```diff
@@ -242,7 +240,7 @@ func (f *Functions) DeleteRecord(profile string, configFileType string, guid str
 	var err error
 	var msg string
 
-	endPointPostfix := "/user/data/" + guid
+	endPointPostfix := commonUtils.FenceDataEndpoint + "/" + guid
```

Comment:
Should this also be a part of `commonUtils`? `commonUtils.FenceDataGuidEndpoint`?

Reply:
uh the `guid` here is an argument, so `FenceDataGuidEndpoint` will just be `FenceDataEndpoint+"/"`